README.md

```diff
@@ -1,65 +1,60 @@
 
-Credentials are a part of our daily lives; driver's licenses are
-used to assert that we are capable of operating a motor vehicle,
-university degrees can be used to assert our level of education,
-and government-issued passports enable us to travel between countries.
-This specification provides a mechanism to express these sorts of
-credentials on the Web in a way that is cryptographically secure,
+Credentials are a part of our daily lives; driver's licenses are used to assert
+that we are capable of operating a motor vehicle, university degrees can be used
+to assert our level of education, and government-issued passports enable us to
+travel between countries. This specification provides a mechanism to express
+these sorts of credentials on the Web in a way that is cryptographically secure,
 privacy respecting, and machine-verifiable.
 
 We encourage contributions meeting the [Contribution
 Guidelines](CONTRIBUTING.md). While we prefer the creation of issues
-and Pull Requests in the GitHub repository, discussions often occur
+and Pull Requests in the GitHub repository, discussions also occur
 on the
 [public-vc-wg](http://lists.w3.org/Archives/Public/public-vc-wg/)
-mailing list as well.
+mailing list.
 
 ## Verifiable Credentials Working Group
 * Group page: [https://www.w3.org/2017/vc/WG/](https://www.w3.org/2017/vc/WG/)
-* charter: [https://www.w3.org/2017/vc/WG/charter.html](https://www.w3.org/2017/vc/WG/charter.html)
+* Charter: [https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html](https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html)
 * Chairs
   * Brent Zundel - @brentzundel
-  * Wayne Chang - @wyc
+  * Kristina Yasuda - @Sakurann
 
 ### Verifiable Credentials github repos
-* [Data Model](https://github.com/w3c/vc-data-model)
 * [Use Cases](https://github.com/w3c/vc-use-cases)
+* [Data Model](https://github.com/w3c/vc-data-model)
+* [Implementation Guide](https://github.com/w3c/vc-imp-guide/)
 * [Data Model Test Cases](https://github.com/w3c/vc-test-suite)
 
-### Other useful links
-* [Public group email archive](https://lists.w3.org/Archives/Public/public-vc-wg/)
+### Discussion Forums
+* [W3C Credentials Community Group Mailing List (Incubation)](https://lists.w3.org/Archives/Public/public-credentials/)
+* [W3C Verifiable Credentials WG Mailing List (Standardization)](https://lists.w3.org/Archives/Public/public-vc-wg/)
 
 ## Process Overview for VC Data Model Pull Requests
-1. For now, we will focus only on merging new errata PRs into this repository,
-   but encourage activity related to new features.
-2. Once a PR is opened, chairs and editors make judgement call on whether
-   changes are substantive or editorial.
+1. Anyone can open a PR on the repository. Note that for the PR to be merged,
+   the individual must agree to the
+   [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy/).
+2. Once a PR is opened on the `v2.0` branch, chairs and editors make judgement
```

Comment:
I expect this may become a source of confusion, because it is not the "usual" setup. That is why I would be in favour of using the 'main' branch for v2.0 for now on.

Reply:
There is no longer a `main` branch. IIRC, we removed it in the v1.1 days because it was confusing people. 

I'm indifferent wrt. if we add it back by renaming the `v2.0` back to `main`.

We probably need to decide if we are going to use tags to specify versions, or branches -- branches make sense when there is active development (such as the v1.1 work), tags make sense when we cut a release and don't plan to update that particular release (which is the case for v1.0 and v1.1). 

So, we could say, `main` is always the latest ED spec. When we need to make a minor release, we create a branch, work on that branch and then release from that branch. Once the release is finalized, we convert the branch to a tag and delete the branch.

This means, we would:

1. Create tags for: `v1.0` and `v1.1`
2. Delete the branches for `v1.0` and `v1.1`
3. Rename the `v2.0` branch to `main` and set Github Pages to publish from `main`.

The other option would just skip step 3 above. Thoughts?

Reply:
Personally, what I am interested in is the v2.0->main change. I am neutral as for (1) and (2); my lazy nature says just leave things as they are, it does not make too much difference in practice.

Reply:
I think having our primary branch be called `main` will be less confusing for folks, especially since we are not working on multiple versions concurrently this time around.

Reply:
> I think having our primary branch be called `main` will be less confusing for folks

The main branch has been renamed to `main`.